src/Symfony/Bundle/FrameworkBundle/Command/DebugAutowiringCommand.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * A console command for autowiring information.
+ *
+ * @author Ryan Weaver <ryan@knpuniversity.com>
+ *
+ * @internal
+ */
+class DebugAutowiringCommand extends ContainerDebugCommand
+{
+    protected static $defaultName = 'debug:autowiring';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDefinition(array(
+                new InputArgument('search', InputArgument::OPTIONAL, 'A search filter'),
+            ))
```

Comment:
this command should have a format option, for people wanting to do some scripting (we generally don't guarantee stability on the text format for usage with |grep`, as this would forbid us to improve DX, but then it requires having other formats)

Reply:
I did this on purpose. I really want this command. But I also realize that it's after feature freeze, and things might change with `debug:container` or `debug:services` in the future. I don't want to encourage (right now) that people use this for scripting. Also, they can use `debug:container --types` and pass a format.